buildSrc/src/main/kotlin/realm-lint.gradle.kts

```diff
@@ -57,6 +57,7 @@ allprojects {
 
         description = "Check Kotlin code style."
         classpath = ktlint
+        jvmArgs = listOf("--add-opens=java.base/java.lang=ALL-UNNAMED")
```

Comment:
why do we need this? for refelction?

Reply:
Some APIs changed or modularized after Java 11, so if you are using Java 17, ktlint will crash. This will fix it.